examples/AspNetCore/README.md

```diff
@@ -0,0 +1,68 @@
+# OpenTelemetry Example Application
+
+This project is an example of ASP.NET Core instrumentation for tracing and
+logging.
+
+The following exporters are configured for viewing the traces:
+
+* Console
+* Zipkin
```

Comment:
Minor: consider alphabetical sorting by default, unless there is a reason not to do so.

Reply:
Only reason was the order in which I added them, I'll get this updated!

Reply:
Updated, along with a couple other lists in this file.

What's general best practice, do I mark the conversation resolved after I apply the change, or will you resolve it after looking at my change?

Reply:
> What's general best practice, do I mark the conversation resolved after I apply the change, or will you resolve it after looking at my change?

Here we're quite flexible, I think people normally resolve the comments without waiting for confirmation (and others can revert it if they have different opinion).
On the opentelemetry-specification repo we've been more careful, normally wait for folks who raised the comment to resolve it.

Reply:
Sounds good, thanks!